base.css

```diff
@@ -30,3 +30,8 @@ body {
 .vscode-body code > div {
     background: none
 }
+
+vscode-body table th,
```

Comment:
This needs to be a class selector: `.vscode-body`

Reply:
Sorry, my mistake.